scarlet2/spectrum.py

```diff
@@ -27,11 +27,11 @@ class StaticArraySpectrum(Spectrum):
     def __init__(self, data, filters, scene):
```

Comment:
One other minor issue: please add ` = eqx.field(static=True)` as the default for `channelindex`.